core/src/poh_recorder.rs

```diff
@@ -462,7 +464,21 @@ impl PohRecorder {
 
     pub fn tick(&mut self) {
         let now = Instant::now();
-        let poh_entry = self.poh.lock().unwrap().tick();
+        let (poh_entry, target_time) = {
+            let mut poh_l = self.poh.lock().unwrap();
+            let poh_entry = poh_l.tick();
+            let target_time = if poh_entry.is_some() {
+                let target_ns_per_tick = PohService::target_ns_per_tick(
+                    self.ticks_per_slot(),
```

Comment:
`ticks_per_slot`, shouldn't ever change in the middle of a run, so might make sense to extract `target_ns_per_tick` into a field in `PohRecorder` as well